ui/src/app/common/services/dataset/dataset.policies.service.ts

```diff
@@ -84,7 +84,7 @@ export class DatasetPoliciesService {
   }
 
   editDataset(datasetItem: Dataset): any {
-    return this.http.put(`${ environment.datasetPoliciesUrl }/${ datasetItem.id }`, datasetItem)
+    return this.http.patch(`${ environment.datasetPoliciesUrl }/${ datasetItem.id }`, datasetItem)
```

Comment:
I'll leave a question here to block this PR from being resolved and merged for now

Reply:
Aborting this pull request